demos/HST/S4_wfc3_template.ecf

```diff
@@ -32,7 +32,7 @@ sum_reads       True    # Sum together all of the reads from one scan?
 
 # Limb-darkening parameters needed to compute exotic-ld
 compute_ld      True
-inst_filter     Prism #filter of JWST/HST instrument, supported list see https://exotic-ld.readthedocs.io/en/latest/views/supported_instruments.html 
+inst_filter     prism #filter of JWST/HST instrument, supported list see https://exotic-ld.readthedocs.io/en/latest/views/supported_instruments.html 
```

Comment:
Why does it need to be lowercase? The website you link to has it as uppercase? It's also really confusing to read that website and realize that you don't need to also add the `JWST_NIRSpec_` part - I think there either needs to be more explanation or change things to just use the format exotic-ld takes

Reply:
The file itself in Sensitivity files uses prism in lowercase. It has been pointed out to me that prism in uppercase doesn't work then. I should open a Github issue for exotic-ld for that. A workaround would be that I make prism lowercase in the generate_LD.py file for now?
My thought there was that you then can't use a throughput file from JWST_NIRSpec when using NIRCam data or some HST file and it's less clunky. But I'm fine with changing it to the full name as used by exotic-ld.

Reply:
Sure, why don't you open an issue there and for now leave it as you've just edited it to be